dcp-0010/dcp-0010.mediawiki

```diff
@@ -0,0 +1,448 @@
+only be applied to <code>mainnet</code>, and current <code>testnet</code>, for
+heights where the new rules are active per the associated on-chain vote.
+
+<!--S_{pow}(h,v) & = & \begin{cases}0 & \text{if } h \le 0,\\S_{full}(1) & \text{if } h = 1,\\\left\lfloor\frac{S_{full}(h) \times 1}{10}\right\rfloor & \text{if } 2 \le h < H_{sv},\\0 & \text{if } h \ge H_{sv} \textbf{ and } v < v_{m},\\\left\lfloor\frac{\left\lfloor\frac{S_{full}(h) \times 1}{10}\right\rfloor \times v}{v_{b}}\right\rfloor & \text{if } h \ge H_{sv} \textbf{ and } v \ge v_{m},\end{cases}-->
+<!--S_{vote}(h) & = & \begin{cases}0 & \text{if } 0 \le x < H_{sv}-1,\\\left\lfloor\frac{S_{full}(h) \times 8}{10 \times v_{b} }\right\rfloor & \text{if } x \ge H_{sv}-1,\end{cases}-->
+<img src="subsidy_proportions.svg" width="100%" height="464" />
+
+Explanation of terms:
+
+S<sub>full</sub> = The full block subsidy at a given height<br />
+S<sub>pow</sub> = The subsidy for proof of work at a given height for a given number of votes<br />
+S<sub>vote</sub> = The subsidy for a single vote at a given height<br />
+h = The height for which the subsidy is to be calculated<br />
+v = The number of votes included in the block
+H<sub>sv</sub> = The height at which stake validation begins (4096 on <code>mainnet</code>)<br />
+v<sub>b</sub> = The number of votes per block (5 on all networks)
+v<sub>m</sub> = The minimum number of required votes (3 on all networks)
+
+===Subsidy Enforcement===
+
+IMPORTANT: The PoS subsidy, and hence each individual  vote subsidy, is based on
+the block PRIOR to the one that contains the vote since that prior block is the
+one being voted on.  In practice, this means the formula to calculate the vote
+subsidy MUST be called with a height of one less than the PoW subsidy formula
+once the height that voting begins at is reached.
+
+The following modified rules MUST be enforced:
+
+* The input value of the one and only input of the coinbase transaction MUST commit to the modified PoW subsidy proportion
+* The total of the output values of the coinbase transaction MUST NOT exceed the modified PoW subsidy proportion plus the total transaction fees
+* The input value of the stakebase of vote transactions MUST commit to the modified single vote subsidy proportion
+* The proportional payouts of vote transactions MUST be calculated based on the modified single vote subsidy proportion
+* The total of the output values of vote transactions MUST NOT exceed modified single vote subsidy proportion
+
+==Rationale==
+
+Proof of Work is intended to provide several key properties to Decred, some of
+which are:
+
+* Unambiguous determination of the exact contents of a block
+* Unforgeable historical timestamps
+* A source of entropy
+* Significantly increased cost to conduct a wide variety of attacks
+* Protection against data manipulation and double spends
+* Contributes to unforgeable costliness and irreversibility
```

Comment:
Sounds like the same as "Significantly increased cost to conduct a wide variety of attacks" unless there's another specific property being alluded to here.

Reply:
Removed.  I did have slightly different things in mind, but you're right to note that it's reasonable to roll them up into "a wide variety of attacks" and I also call these properties out more specifically later.